### PR TITLE
Homework 5

### PR DESCRIPTION
**Pull request by Tailor-AI**
A lot of new stuff to take in. Dont know if I understood the marry task correct.

### CODE REVIEW

File: ShoppingListApi/Controllers/PersonController.cs [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+
+            if (oldPerson == null)
+            {
+                return NotFound($"Person by id {id} was not found.");
+            }
+
+            oldPerson.Pets = person.Pets;
+            oldPerson.Name = person.Name;
+            oldPerson.Age = person.Age;
+            oldPerson.SpouseId = person.SpouseId;
```
Comment (Almantask) [inline ShoppingListApi/Controllers/PersonController.cs]: Out of place. Move this to a person class.

File: ShoppingListApi/Controllers/PersonController.cs [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+            }
+
+            return Ok();
+        }
+
+        private void Marry(Person person1, Person person2)
+        {
+            person1.SpouseId = person2.Id;
+            person2.SpouseId = person1.Id;
+        }
```
Comment (Almantask) [inline ShoppingListApi/Controllers/PersonController.cs]: Move this to a person class.

**Review by Almantask — CHANGES_REQUESTED** [review]

File: ShoppingListApi/Controllers/PersonController.cs (hunk 1 above)
Comment (Tailor-AI) [reply ShoppingListApi/Controllers/PersonController.cs]: Moved this logic to Person class.

File: ShoppingListApi/Controllers/PersonController.cs (hunk 2 above)
Comment (Tailor-AI) [reply ShoppingListApi/Controllers/PersonController.cs]: Moved to Personclass.

**Review by Almantask — APPROVED** [review]

**Pull request closed without merging**